### DISCUSSION TITLE
Compatibility with Apple devices Questions?  why ios safari  does not support llhls

### DISCUSSION THREAD

**Discussion by fastfading** [bluenviron/mediamtx#2288] [Questions]
In this part
Compatibility with Apple devices
https://github.com/bluenviron/mediamtx#compatibility-with-apple-devices

1. what if I set
hlsEncryption: no 
but use a netscaler to handle http to https forwarding .  
will safari support llhls in this case  ? 

2. "Keep also in mind that not all H264 video streams can be played on Apple Devices due to some intrinsic properties (distance between I-Frames, profile). "
could you tell us more about the limitation ? 

we have met a issue that , llhls is not running properly on ios device ,  I am not sure why ?

**Comment by aler9** [comment]
open an issue and provide relevant data that allows to replicate the problem, including a sample file.
